jackson/src/main/resources/reference.conf

```diff
@@ -1,40 +1,25 @@
 play.modules.enabled += com.lightbend.lagom.javadsl.jackson.JacksonModule
 
-lagom.serialization.json {
-  
-  #//#jackson-modules
-  # The Jackson JSON serializer will register these modules.
-  # It is also possible to use jackson-modules = ["*"] to dynamically
-  # find and register all modules in the classpath.  
-  jackson-modules += "com.fasterxml.jackson.module.paramnames.ParameterNamesModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.jdk8.Jdk8Module"
-  jackson-modules += "com.fasterxml.jackson.datatype.jsr310.JavaTimeModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.pcollections.PCollectionsModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.guava.GuavaModule"
-  #//#jackson-modules  
-  
-  # The serializer will compress the payload if the message class
-  # implements the CompressedJsonable marker interface and the payload 
-  # is larger than this value. Only used for remote messages within 
-  # the cluster of the service.
-  compress-larger-than = 1024b
-  
-  # Define data migration transformations of old formats to current 
-  # format here as a mapping between the (old) class name to be
-  # transformed to the JacksonJsonMigration class that implements
-  # the transformation.  
-  migrations {
-  }
-}
-
 akka.actor {
   serializers {
-    lagom-json = "com.lightbend.lagom.internal.jackson.JacksonJsonSerializer"
+    # Placeholder for Lagom 1.5.x JacksonSerializer.
+    # Needed for deserialization of messages during rolling update, and old data (events, snapshots)
+    # that were serialized with the Lagom 1.5.x JacksonSerializer.
+    # Note that by default there is no
+    old-lagom-json = "com.lightbend.lagom.internal.jackson.OldJacksonJsonSerializer"
   }
   serialization-bindings {
-    "com.lightbend.lagom.serialization.Jsonable" = lagom-json
+    # Bind Jsonable to Akka JacksonJsonSerializer
+    "com.lightbend.lagom.serialization.Jsonable" = jackson-json
   }
   serialization-identifiers {
-    "com.lightbend.lagom.internal.jackson.JacksonJsonSerializer" = 1000002
+    # For Lagom 1.5.x JacksonSerializer.
+    "com.lightbend.lagom.internal.jackson.OldJacksonJsonSerializer" = 1000002
   }
 }
+
+akka.serialization.jackson {
+  jackson-modules += "com.fasterxml.jackson.datatype.pcollections.PCollectionsModule"
+  jackson-modules += "com.fasterxml.jackson.datatype.guava.GuavaModule"
+  # FIXME do we have to remove the DefaultScalaModule?
```

Comment:
ping @marcospereira

Reply:
> afterburner seems to make it too easy for users to get a serialiser for free

Unless I have missed something that is a misunderstanding. Afterburner is only supposed to be a transient performance optimization.

However, I have removed it because I couldn't see any improvement of using it: https://github.com/akka/akka/pull/27064

Reply:
The question about `DefaultScalaModule` remains. It's easy to disable with config, but on the other hand it might not hurt to have it included?